spec/actors/hyku_addons/actors/task_master_work_actor_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
```

Comment:
Looks like this spec could be removed in favor of `spec/actors/hyku_addons/actors/task_master/work_actor_spec.rb` above.